airflow/providers/google/cloud/hooks/workflows.py

```diff
@@ -212,7 +212,7 @@ def list_workflows(
 
         :param filter_: Filter to restrict results to specific workflows.
         :type filter_: str
-        :param order_by: Comma-separated list of fields that that
+        :param order_by: Comma-separated list of fields that
             specify the order of the results. Default sorting order for a field is ascending.
```

Comment:
```suggestion
            specifies the order of the results. Default sorting order for a field is ascending.
```

While we’re at this…